src/panels/config/core/ha-config-core-form.ts

```diff
@@ -223,6 +249,7 @@ class ConfigCoreForm extends LitElement {
       await saveCoreConfig(this.hass, {
         latitude: location[0],
         longitude: location[1],
+        currency: this._currencyValue,
```

Comment:
```suggestion
        currency: this._currencyValue || "",
```
This will send a blank string if the value is `null` (None)

Reply:
Thanks ❤️

Reply:
Shouldn't we just send `null`?

Reply:
To allow that, change the backend to:
```suggestion
        vol.Optional(CONF_CURRENCY): vol.Any(str, None),
```

Reply:
vol.Optional = None?

Reply:
None, Optional is that it doesn't have to be present, but if it is, it should be a string.